main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: car
+//  action 1: the car turns
+car.turns();
+//  action 2: the car accelerate
+car.accelerate();
+//  action 3: the car stop
+car.stop();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: computer
+//  action 1: the computer suspend
+computer.suspend();
+//  action 2: the computer hibernate
+computer.hibernate();
+//  action 3: the computer shutDown
+computer.shutDown();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: oven
+//  action 1: the oven heats
+oven.getHeat();
+//  action 2: the oven colds
+oven.getCold();
+//  action 3: the oven keep temperature
+oven.keepTemperature();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: pet
+//  action 1: the pet eat
+pet.eat();
+//  action 2: the pet sleep
+pet.sleep();
+//  action 3: the pet play
+pet.play();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: TV
+//  action 1: the TV change the channel
+tv.changeChannel();
+//  action 2: the TV change the source
+tv.changeSource();
+//  action 3: the TV standby
+tv.standby();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: elevator
+//  action 1: the elevator go up
+elevator.goUp();
+//  action 2: the elevator go down
+elevator.goDown();
+//  action 3: the elevator open the door
+elevator.openDoor();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: air conditioner
+//  action 1: the air conditioner check the temperature
+airConditioner.checkTemperature();
+//  action 2: the air conditioner cold the air
+airConditioner.cold();
+//  action 3: the air conditioner ventilate 
+airConditioner.ventilate();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.